Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -267,7 +267,9 @@ open class MapView: UIView {
             return
         }
 
-        if needsDisplayRefresh {
+        if needsDisplayRefresh
+            || cameraAnimatorsSet.allObjects.count > 0
+            || !pendingAnimatorCompletionBlocks.isEmpty {
```

Comment:
Good catch! We had a similar issue with pending completion blocks < 10.0.0, and required a similar fix IIRC.